proto/process/agent.proto

```diff
@@ -910,6 +910,7 @@ message HTTPStats {
 	message Data {
 		uint32 count = 1;
 		bytes latencies = 2;
+		uint64 firstLatencySample = 3;
```

Comment:
How about using `oneof` to indicate that only latencies or firstLatencySample will be set ?

Reply:
Talked IRL about this. While semantically better this would result in an interface type that would have its own set of issues.